metabase/metabase
# Update copy in confirm modal for cancelling new model

### How to Test

1. `+ New`
2. `Model`
3. `Native Query`
4. `Cancel`

You should see a Confirm modal with the following copy:

<img width="1512" alt="image" src="https://user-images.githubusercontent.com/380816/195679905-6baec484-91d2-4aab-ad37-923caeb431d3.png">
